source/core/lethe_grid_tools.cc

```diff
@@ -11,8 +11,6 @@
 #include <deal.II/grid/manifold_lib.h>
 
 #include <cmath>
-#include <unordered_map>
-
```

Comment:
Why remove this include? Unordered map are used in the lethe_grid_tools. This was required for ARM compilation.

Reply:
We don't have any unordered_maps in the lethe_grid_tools anymore. is it necessary?

Reply:
if we don't then it's cool. just make sure all includes are there.